From: Benda Xu <orv@debian.org>
Date: Sat, 9 Mar 2024 17:37:03 +0800
Subject: config/Substfiles.am: restrict to the dedicated list.

We recover the scope of $(SUBSTFILES) to explicitly control what files
are being generated from the corresponding .in.

put zfs_prepare_disk.8 into substitution list. b53077a9e751 forgot to
include the zfs_prepare_disk.8 in the substitution list, although it
built in the past because 38e2e9ce8327 accidentally dropped the
$(SUBSTFILES) from the "%: %.in" pattern.

Signed-off-by: Benda Xu <orv@debian.org>
Forwarded: https://github.com/openzfs/zfs/pull/15980
---
 config/Substfiles.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config/Substfiles.am b/config/Substfiles.am
index 18422bf..2459637 100644
--- a/config/Substfiles.am
+++ b/config/Substfiles.am
@@ -44,4 +44,4 @@ SUBSTFILES =
 CLEANFILES += $(SUBSTFILES)
 dist_noinst_DATA += $(SUBSTFILES:=.in)
 
-$(call SUBST,%,)
+$(SUBSTFILES): $(call SUBST,%,)
